wireapp/wire-desktop
# fix: Recreate shortcut when updating on Windows (SQCORE-919)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

- after updating on Windows, an old version of Wire was started

### Causes (Optional)

- shortcut on Desktop was pointing to the .exe file in the subfolder (e.g. `%AppData%/wire/wire-3.26.1234/wire-3.26.1234.exe`)
- this led to an old version being started

### Solutions

- link the shortcut to the main .exe file (e.g. `%AppData%/wire/Wire.exe`)

### Testing

- install old version
- install new version
- check the Desktop's shortcut target

#### PR Post Merge Checklist for internal contributors

----
